src/models/newsletters.ts

```diff
@@ -46,38 +46,33 @@ const baseNewsletterModel = {
 	brazeSubscribeAttributeNameAlternate: optional(t.array(t.string)),
 	brazeSubscribeEventNamePrefix: optional(t.string),
 	brazeNewsletterName: optional(t.string),
-	emailEmbed: EmailEmbedType,
+	emailEmbed: EmailEmbedCodec,
 	restricted: t.boolean,
 	paused: t.boolean,
 	emailConfirmation: t.boolean,
 	signupPage: optional(t.string),
 	exampleUrl: optional(t.string),
-	illustration: optional(NewsletterIllustrationType),
+	illustration: optional(NewsletterIllustrationCodec),
 	campaignName: optional(t.string),
 	campaignCode: optional(t.string),
 };
 
-export const CancelledEmailNewsletterType = t.type({
-	...baseNewsletterModel,
-	cancelled: t.literal(true),
-});
```

Comment:
The changes to the model will break the "preview from CSV" feature, but that's only a convenience for local dev work. The preview feature can be fixed on another PR.

Reply:
That's a good point! I'm concerned the tests didn't fail in this case though. Is there a simple change that would have failed the pipeline?